### PR TITLE
Please review - feb 23 client meeting agenda

### PR DESCRIPTION
**Pull request by katmartel**
Please note: After the client suggested Ant Design's notification component instead of alerts, I figured I would test switching over to notifications after the meeting. Since it was an easy fix, I thought that I might as well replace all of the alerts with the notifications. I hope that's alright to include in this PR.

Also, please review the points under 'Action Items' in my meeting notes and provide your thoughts in a comment. @jpabadir @aleksssk @wrc9817

### CODE REVIEW

**Review by aleksssk — APPROVED** [review]
Looks good

**Review by wrc9817 — APPROVED** [review]

File: meeting-minutes/client-minutes/feb23-client-minutes.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Creating matches with Mentees and Mentors from common organizations.
+- Switching to the notification component from the pop-ups.
+
+Pace A then ran their application to show the new nav bar they created and the ability to only access the Auth page when logged in.
+
+Pace B demonstrated what they worked on, and they modified their admin and mentee forms, as well as the ability to send emails.
+
+### ACTION ITEMS
+
+2/24-01. If Codebar does decide to clone our repository to work on what we've done after April, does everyone on our team give their consent? A reminder that we signed the IP Agreement at the beginning of the course, and so I don't think we can object to this. However, the client asked if our team would be alright with Codebar forking our repo (if we're chosen out of the two groups), and so I want to make sure I have everyone's input before I agree or disagree. We should probably ask the instructor about this as well, since we signed for Pace, and I'm not sure if there's surrounding legalities of another organization using the project.
```
Comment (jpabadir) [inline meeting-minutes/client-minutes/feb23-client-minutes.md:59]: I give my full consent to any partners of pace to use this repo in whatever way they want.

File: meeting-minutes/client-minutes/feb23-client-minutes.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Given that there's only 6 weeks left of the course, I mentioned to the client that it may not be feasible to incorporate new functionality, especially since we want to ensure that what we've created is functioning 100% as expected.
+Although it would be great if we could incorporate those things, she seemed to understand that we don't have a lot of time remaining. Technically, once we're able to associate embeddable mentee forms with the Organization (and create matches based on that) all minimum viable product functionality for the project will have been met. Therefore, anything else is considered as bonus. We should discuss as a team how we choose to proceed.
+
+Stylistic wise, the client doesn't have any preferences, since the organization taking over the application might refactor that completely. However, she does prefer Ant Design's notification component over the pop-ups.
+
+From our conversations with the client, the priority seems to be:
+
+- Finalizing the iFrame form so that it's associated with the organization.
+- Creating matches with Mentees and Mentors from common organizations.
+- Switching to the notification component from the pop-ups.
```
Comment (jpabadir) [inline meeting-minutes/client-minutes/feb23-client-minutes.md:51]: Noted.

**Review by jpabadir — APPROVED** [review]
Great work! Will work to match our dev priorities to the client's in the last weeks of the class. Good initiative to go ahead and replace those ugly warnings with proper ones.

**Pull request merged**